mithril-signer/src/configuration.rs

```diff
@@ -76,34 +77,39 @@ pub struct Configuration {
 
 impl Configuration {
     /// Return the CardanoNetwork value from the configuration.
-    pub fn get_network(&self) -> Result<CardanoNetwork, ConfigError> {
+    pub fn get_network(&self) -> StdResult<CardanoNetwork> {
         CardanoNetwork::from_code(self.network.clone(), self.network_magic)
-            .map_err(|e| ConfigError::Message(e.to_string()))
+            .with_context(|| "Could not read Network from configuration.")
```

Comment:
The network name and magic would be useful in the context message

Reply:
my bad, I thought it did not implement `Display`, but it does!